syntax = "proto3";

package api;

message ApplicationRegisterRequest {
  string name = 1;
}

message ApplicationUpdateRequest {

}

message ApplicationDeleteRequest {

}

message ApplicationQueryRequest {

}

message ApplicationListRequest {
  int64 last_create_time = 1;
}

message ApplicationQueryResponse {
  string id = 1;
  string name = 2;
  int64 create_time = 3;
}

message ApplicationListResponse {
  repeated ApplicationQueryResponse applications = 1;
  int32 total = 2;
}
